File: java/fundamentos/mostrar_informacion_completa.java
package fundamentos;
public class mostrar_informacion_completa {
    public static void main(String[] args) {
        String nombre1 = "Ana Gómez";
        String telefono1 = "3001112233";
        String correo1 = "ana@gmail.com";
        String direccion1 = "Calle 10 #20-30";
        String nombre2 = "Luis Pérez";
        String telefono2 = "3012223344";
        String correo2 = "luis@gmail.com";
        String direccion2 = "Carrera 15 #25-40";
        System.out.println("=== INFORMACIÓN COMPLETA DE CONTACTOS ===");
        System.out.println("\nContacto 1:");
        System.out.println("Nombre: " + nombre1);
        System.out.println("Teléfono: " + telefono1);
        System.out.println("Correo: " + correo1);
        System.out.println("Dirección: " + direccion1);
        System.out.println("\nContacto 2:");
        System.out.println("Nombre: " + nombre2);
        System.out.println("Teléfono: " + telefono2);
        System.out.println("Correo: " + correo2);
        System.out.println("Dirección: " + direccion2);
    }
}

File: visual_basic/fundamentos_del_lenguaje/mostrar_informacion_completa/mostrar_informacion_completa.vb
Imports System
Module mostrar_informacion_completa
    Sub Main(args As String())
        Dim nombre As String = "Juan Pérez"
        Dim telefono As String = "3001234567"
        Dim correo As String = "juan@email.com"
        Dim direccion As String = "Calle 123 #45-67"
        Console.WriteLine("=== INFORMACIÓN COMPLETA ===")
        Console.WriteLine("Nombre: " & nombre)
        Console.WriteLine("Teléfono: " & telefono)
        Console.WriteLine("Correo: " & correo)
        Console.WriteLine("Dirección: " & direccion)
        Console.ReadLine()

    End Sub
End Module